packages/fxa-settings/src/components/Settings/GetDataTrio/index.tsx

```diff
@@ -29,6 +29,30 @@ export type GetDataTrioProps = {
   onAction?: (type: 'download' | 'copy' | 'print') => void;
 };
 
+export const GetDataCopySingleton = ({ value, onAction }: GetDataTrioProps) => {
+  return (
+    <Localized id="get-data-trio-copy" attrs={{ title: true }}>
+      <button
+        title="Copy"
+        type="button"
+        onClick={async () => {
+          const copyValue = Array.isArray(value) ? value.join('\r\n') : value;
+          await copy(copyValue);
+          onAction?.('copy');
+        }}
+        data-testid="databutton-copy"
+        className="w-12 h-12 relative inline-block text-grey-500 rounded active:text-blue-600 focus:outline focus:outline-2 focus:outline-offset-2 focus:outline-blue-500 hover:bg-grey-50"
+      >
+        <CopyIcon
```

Comment:
The icons might need aria-labels since the buttons don't have text? I'm not entirely sure about this, but I think the 'title' tag on the button only adds a tooltip and might not be read by a SR.

Also a bit of an aside, I noticed that the download button is a link instead of a button (I know that wasn't your addition though, so might not be in the scope of this ticket!).

Reply:
I was also surprised by that `<a>`! I'll add those aria labels on!